docs/Schema-Model.md

```diff
@@ -0,0 +1,161 @@
+i:decnr: 15.468259749904854
+i:diffmaglim: 29.968622
+i:distnr: 8.077449
+i:distpsnr1: 29.968622
+i:distpsnr2: 29.968622
+i:distpsnr3: 29.968622
+i:drb: 0.999752
+i:drbversion: d6_m7
+i:dsdiff: -1.18682505E29
+i:dsnrms: 9.464858
+i:elong: 1.8730389
+i:exptime: 29.968622
+i:fid: 2
+i:field: 568
+i:fwhm: 1.8730389
+i:isdiffpos: t
+i:jd: 2482047.4790935214
+i:jdendhist: 2482047.4790935214
+i:jdendref: 2482047.478571366
+i:jdstarthist: 2482047.4790935214
+i:jdstartref: 2482047.4772142665
+i:magap: 29.968622
+i:magapbig: 29.968622
+i:magdiff: -1.1868114E29
+i:magfromlim: 0.46825972
+i:maggaia: 29.968622
+i:maggaiabright: -1.1868139E29
+i:magnr: 29.968622
+i:magpsf: 29.968622
+i:magzpsci: 29.968622
+i:magzpscirms: 0.037692007
+i:magzpsciunc: 7.1450763E-6
+i:mindtoedge: 673.5927
+i:nbad: 0
+i:ncovhist: 239
+i:ndethist: 1
+i:neargaia: 29.968622
+i:neargaiabright: -1.1868139E29
+i:nframesref: 15
+i:nid: 1034
+i:nmatches: 769
+i:nmtchps: 3
+i:nneg: 3
+i:objectId: ZTF19acmbtud
+i:objectidps1: 139540932695408573
+i:objectidps2: 139540932846403517
+i:objectidps3: 139540932695408573
+i:pdiffimfilename: ztf_20191101527951_000568_zr_c04_o_q4_scimrefdiffimg.fits
+i:pid: 1029714075307
+i:programid: 1
+i:programpi: Kulkarni
+i:publisher: Fink
+i:ra: 128.2831039126012
+i:ranr: 128.28545567346546
+i:rb: 0.774291
+i:rbversion: t17_f5_c3
+i:rcid: 15
+i:rfid: 569360317
+i:schemavsn: 3.3
+i:scorr: 11.468259749904854
+i:seeratio: 1.8730389
+i:sgmag1: -1.1868139E29
+i:sgmag2: 29.968622
+i:sgmag3: 29.968622
+i:sgscore1: 0.0
+i:sgscore2: 0.99584574
+i:sgscore3: 0.5
+i:sharpnr: -1.18682505E29
+i:sigmagap: 0.46825972
+i:sigmagapbig: 0.46825972
+i:sigmagnr: 0.216
+i:sigmapsf: 0.15521143
+i:simag1: -1.1868139E29
+i:simag2: 29.968622
+i:simag3: 29.968622
+i:sky: 0.5201477
+i:srmag1: 29.968622
+i:srmag2: 29.968622
+i:srmag3: -1.1868139E29
+i:ssdistnr: 0.0
+i:ssmagnr: 29.968622
+i:ssnamenr: 52124
+i:ssnrms: 13.6383
+i:sumrat: 1.8730389
+i:szmag1: -1.1868139E29
+i:szmag2: 29.968622
+i:szmag3: -1.1868139E29
+i:tblid: 22
+i:tooflag: 0
+i:xpos: 2398.9841
+i:ypos: 2400.76
+i:zpclrcov: -1.18682505E29
+i:zpmed: 29.968622
+d:cdsxmatch: Unknown
+d:rfscore: 0.0
+```
+
+The above row will be translated into graph data model as follows:
+
+### Vertices
+- Every alert (single row in hbase) will be modelled as a graph vertex.
+- Following alert properties will be modelled as vertex labels
+    - Supernova Random Forest classification ```(d:rfscore)```
+    - Supernova Bayesian Neural Network classification ```(d:snn)``` [coming soon]
+    - Microlensing Random Forest classification ```(d:mulens_class_i)``` [coming soon]
+    - Cross-match label ```(d:cdsxmatch)```
+    - Star/Galaxy separation ```(i:classtar)```
+    - The object ID itself if we use individual alerts ```(i:objectId)```
+
+### Edges
+- Create an edge between all the vertices that have same properties
+- So if Vertex 1 has ```(d:rfscore)``` property and Vertex 5 also has the same property, we will create an edge between them.
```

Comment:
We could also have a 'similarity' edge that summarizes all edges between two vertices. To be more clear, if two vertices are connected by 4 edges (4 properties "in common"), the 'similarity' edge would be strength 4.